docs/docs/spec.md

```diff
@@ -771,18 +792,15 @@ hold:
 
 `T` has nullness-subtype-establishing direct-supertype edges to the following:
 
--   if `T` is an augmented [intersection type]: all the intersection type's
-    elements whose [nullness operator] is `NO_CHANGE` or `MINUS_NULL`
+-   if `T` is an augmented [intersection type]: each of the intersection type's
+    elements whose [nullness operator] we are not [worried] is `UNION_NULL`
 
--   if `T` is an augmented type variable: all the corresponding type parameter's
-    upper bounds whose nullness operator is `NO_CHANGE` or `MINUS_NULL`
+-   if `T` is an augmented type variable: each of the corresponding type
+    parameter's upper bounds whose nullness operator we are not worried is
+    `UNION_NULL`
```

Comment:
OK, this sounds a little convoluted: "each of X whose nullness operator we are not worried is `UNION_NULL`"

What about "each of X, except those whose nullness operator we are worried is `UNION_NULL`"? I don't know if that's much better.